streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -554,9 +554,9 @@ public final <K, V> void addSink(final String name,
         nodeGroups = null;
     }
 
-    public final <KIn, VIn, KOut, VOut> void addProcessor(final String name,
-                                                          final ProcessorSupplier<KIn, VIn, KOut, VOut> supplier,
-                                                          final String... predecessorNames) {
+    public final void addProcessor(final String name,
```

Comment:
Side cleanup. We don't get (and don't need) any type verification here. So removing unnecessary generic types